source/patterns/@aws-solutions-constructs/aws-lambda-step-function/lib/index.ts

```diff
@@ -20,9 +20,9 @@ import { Construct } from '@aws-cdk/core';
 import * as logs from '@aws-cdk/aws-logs';
 
 /**
- * @summary Properties for the LambdaToStepFunction class.
+ * @summary Properties for the LambdaToStepFunctions class.
  */
-export interface LambdaToStepFunctionProps {
```

Comment:
We don't want to change anything in the old interface, if we change the name of the props interface and the construct class we will break any existing customer code.